### ISSUE TITLE
[update] EmoNavi

### ISSUE THREAD

**Issue by muooon** [kozistr/pytorch_optimizer#454]
[update] (https://github.com/muooon/EmoNavi/tree/dev) v3.3 

EmoNavi v3.3 has been released.  (automatic learning rate) 

Please take a look when you have time, and if needed, feel free to incorporate it into PyTorch-Optimizer.
(We sincerely appreciate your consistent, meticulous updates to various optimizers. Discovering new optimizers and being able to start using them easily is truly appealing. We look forward to future updates and the ongoing development of this project. Thank you.)

EmoNavi v3.3 をリリースしました (完全自動学習率)

お時間のあるときにご覧頂いて、必要あればぜひ PyTorch-Optimizer に取り入れてください。
(さまざまな optimizer を、いつも丁寧に更新してくださり感謝します。新しい optimizer との出会い、そして簡単に使い出せることが魅力的です。今後の更新も含め、このプロジェクトについて期待しています。ありがとうございます)
